src/applications/vaos/appointment-list/hooks/useHandleClick.js

```diff
@@ -8,10 +10,8 @@ export default function useHandleClick({ link, idClickable }) {
   return useCallback(
     () => {
       return () => {
-        if (!window.getSelection().toString()) {
-          focusElement(`#${idClickable}`);
-          history.push(link);
-        }
```

Comment:
I'm not sure why this check was here but I don't see the need to check for selected text.